src/utils.ts

```diff
@@ -113,9 +113,49 @@ function readCompilerOptions(configPath: string) {
   return parsedConfig.options;
 }
 
+function getStartDir(): string {
+  // This is needed because of the way our tests are structured.
+  // If this is being executed as a library (under node_modules)
+  // we want to start with the project directory that's three
+  // levels above.
+  // If t his is being executed from the test suite, we want to start
+  // in the directory of the test
+
+  const grandparent = path.resolve(__dirname, '..', '..');
+  if (grandparent.endsWith('/node_modules')) {
+    return path.resolve(grandparent, '..');
+  }
+  return '.';
+}
+
+function getPathToClosestTSConfig(
+  startDir?: string,
```

Comment:
Instead of making this nullable, how about we just add a default parameter of getStartDir() - I'm reasonably sure we can use any js expression as a default value.

Reply:
By doing it this way, we account for the case when nothing is passed in, making this easier to use. If we set a default value, we'll have to check for null/undefined because, theoretically, such a value could be passed in

Reply:
That's a fair point.